### PR TITLE
Feature dynamic lights

### LINKED ISSUE #394: Dynamically place lights (Task from last week) - Add dynamic lighting entity creation to rooms
Lighting entities should be created either on the walls or the roof depending on the position of the light source.

This is a subtask of #289 - Dynamically place lights (Task from last week)

### PR DESCRIPTION
**Pull request by Diocruel**
A bit late, but here are **_dynamic_** lights. 

All the methods and classes needed for dynamic light placement. It is fully tested, except for the light place methods in RoomNode and RoomLevelFactory, as they will only be reached if the constant of renderTorches is set to true. Since I was not sure to make this a real final constant, I kept it as a normal public static field.

The coverage of the extra lines is above 80%

Credit to where it is due:
@Taeir bug fixing 
@ChielBruin for light loading
@WytzeElhorst for the models. (which are already on master)

This closes #394.

### CODE REVIEW

File: src/main/java/nl/tudelft/contextproject/model/entities/VRPlayer.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
 public class VRPlayer extends MovingEntity implements PhysicsObject, TickProducer, Health {
 
 	//Physics interaction constants.
-	public static final float JUMP_SPEED = 7f;
+	public static final float JUMP_SPEED = 15f;
```
Comment (Taeir) [inline src/main/java/nl/tudelft/contextproject/model/entities/VRPlayer.java]: Jump speed should be restored to 7f.

File: src/main/java/nl/tudelft/contextproject/Main.java [hunk 2]
```diff
@@ -99,7 +99,7 @@ public static void main(String[] args) {
 		
 		//Set frustum distances here before app starts
 		main.preconfigureFrustrumNearFar(0.1f, 512f);
-		
+		//main.pre
```
Comment (Taeir) [inline src/main/java/nl/tudelft/contextproject/Main.java]: Should be removed

**Comment by Taeir** [comment]
From darkness, into light.

Looking good. Apart from the 2 minor things I mentioned, I think this can be merged.

**Comment by ChielBruin** [comment]
Looks good! when @Taeir's comments are resolved it can be merged!

**Pull request merged**